haystack/modeling/model/prediction_head.py

```diff
@@ -19,11 +19,11 @@
 logger = logging.getLogger(__name__)
 
 
-try:
-    from apex.normalization.fused_layer_norm import FusedLayerNorm as BertLayerNorm
-except (ImportError, AttributeError) as e:
-    logger.debug("Better speed can be achieved with apex installed from https://www.github.com/nvidia/apex .")
-    BertLayerNorm = torch.nn.LayerNorm
+# try:
+#     from apex.normalization.fused_layer_norm import FusedLayerNorm as BertLayerNorm
```

Comment:
I guess the `BertLayerNorm` was not used; I would rather delete these lines than just comment them.